packages/jest-environment-enzyme/package.json

```
@@ -41,12 +41,15 @@
     "url": "https://github.com/FormidableLabs/enzyme-matchers/issues"
   },
   "peerDependencies": {
-    "enzyme": "3.x",
-    "jest": ">=22.0.0",
-    "react": "^0.13.0 || ^0.14.0 || ^15.0.0 || >=16.x"
+    "enzyme": "^3.4.0",
+    "jest": "^22.0.0 || ^23.0.0 || ^24.0.0 || ^25.0.0",
+    "react": "^0.13.0 || ^0.14.0 || ^15.0.0 || ^16.0.0"
   },
   "devDependencies": {
-    "cpy-cli": "^2.0.0"
+    "cpy-cli": "^2.0.0",
+    "enzyme": "^3.4.0",
+    "jest": "^25.2.0",
+    "react": "^16.5.1"
```

dep ranges for peer deps should be identical:
```suggestion
    "enzyme": "^3.4.0",
    "jest": "^22.0.0 || ^23.0.0 || ^24.0.0 || ^25.0.0",
    "react": "^0.13.0 || ^0.14.0 || ^15.0.0 || ^16.0.0"
```

---

Dev dependencies can be a range?

---

What does it mean for a dev dependency to be a range? I've never seen this before.

---

The same as for a peer dep - it uses the latest that suffices, but if there’s an existing version it’ll use that.

Note that ^ is a range too, as is >=.